app-frontend/src/app/core/services/auth.service.js

```diff
@@ -20,6 +22,8 @@ export default (app) => {
             if (!APP_CONFIG.error) {
                 this.initAuth0(APP_CONFIG);
             }
+
+            this.pendingReauth = null;
```

Comment:
This gets assigned a promise so I didn't want to mix types. It doesn't have to be a promise, it could also be `true`/`false` but at some point we might want to cancel the pending reauthorization so it could be handy to have it available.